lib/request.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Parse params found in path of the request URL
+ *
+ * @param {Request} request
+ * @returns {{proofSetId?: string, pieceCid?: string}}
+ */
+export function parseRequest (request) {
+  const url = new URL(request.url)
+  const [proofSetId, pieceCid] = url.pathname.split('/').filter(Boolean)
```

Comment:
will need to be updated once @NikolasHaimerl set up the new domains

Reply:
Can you explain why? I'm not sure if domain name would affect this function unless we change how we construct the URL.